frameworks/helloworld/tests/test_sanity.py

```diff
@@ -261,8 +261,3 @@ def fn():
     # Verify ZK is unchanged
     zk_config_new = shakedown.get_zk_node_data(zk_path)
     assert zk_config_old == zk_config_new
-
-@pytest.mark.upgrade
-@pytest.mark.sanity
-def test_upgrade_downgrade():
-    sdk_test_upgrade.upgrade_downgrade(PACKAGE_NAME, PACKAGE_NAME, DEFAULT_TASK_COUNT)
```

Comment:
Is this expected to fail as of this PR? If so the test should probably just be disabled instead of deleted